validation/validation_test.go

```diff
@@ -88,13 +88,44 @@ func TestValidateBasic(t *testing.T) {
 	assert.Nil(t, runtimeInsideValidate(g))
 }
 
+// Test whether rootfs Readonly can be applied or not
+func TestValidateRootFS(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetRootReadonly(true)
+
+	assert.Nil(t, runtimeInsideValidate(g))
+}
+
+// Test whether hostname can be applied or not
+func TestValidateHostname(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetHostname("hostname-specific")
+
+	assert.Nil(t, runtimeInsideValidate(g))
+}
+
+// Test whether mounts are correctlt mounted
+func TestValidateMounts(t *testing.T) {
+	// TODO mounts generation options have not been implemented
```

Comment:
we can add this test when 'mounts generate' will be done.

Reply:
Added such TODO comment